### ISSUE TITLE
configure logger to write to a non-stdout file

### ISSUE THREAD

**Issue by laser** [filecoin-project/rust-fil-proofs#624]
### Description

rust-fil-proofs is a library, but its only consumer is go-filecoin. As such, Rust logging norms (e.g. libraries shouldn't log to stdout themselves) are hard to follow. The rust-fil-proofs (library) logger can't write to the go-filecoin (daemon) logger, so instead we initialize the logger ourselves and write to stdout. 

**No coordination exists between the `libfilecoin_proofs.a` consumer's logger and the loggers configured in `libfilecoin_proofs.a`, so log messages will be interleaved if they're all sent to stdout.** If JSON logging is enabled, interleaving writes to stdout can result in malformed (e.g.) JSON.

To work around this issue, we should introduce a mechanism (environment variable, probably) which can be used to control the location to which rust-fil-proofs log entries will be written.

### Acceptance criteria

- environment variable controls log output path
- environment variable is documented somewhere
- defaults to stdout (e.g. `/dev/stdout`)

**Comment by porcuquine** [comment]
If we use an env var to communicate, let's also add it to the config, which will help internally document available options. This just means the var will need a `FIL_PROOFS_` prefix, which seems okay.
